stereo_image_proc/CMakeLists.txt

```diff
@@ -35,6 +56,60 @@ rclcpp_components_register_node(${PROJECT_NAME}
   EXECUTABLE point_cloud_node
 )
 
+if(DEFINED ROS_VITIS)
+
+  set(lib_depends
+  "rclcpp"
+  "rclcpp_components"
+  "vitis_common"
+  )
+  # disparity_fpga component (with OpenCL)
+  ament_auto_add_library(disparity_fpga SHARED src/stereo_image_proc/disparity_fpga.cpp)
+  ament_target_dependencies(${PROJECT_NAME} ${lib_depends})
+  target_include_directories(disparity_fpga PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include/stereo_image_proc>
+    $<INSTALL_INTERFACE:include>
+    $ENV{XILINX_HLS}/common/technology/autopilot
+    $ENV{XILINX_HLS}/include)
+  target_link_libraries(disparity_fpga ${OpenCL_LIBRARY})
+  target_compile_definitions(disparity_fpga PRIVATE "COMPOSITION_BUILDING_DLL")
+
+  rclcpp_components_register_nodes(disparity_fpga "stereo_image_proc::DisparityNodeFPGA")
```

Comment:
Why is this macro call differing from the one above ? 
https://github.com/ros2/rclcpp/blob/master/rclcpp_components/cmake/rclcpp_components_register_node.cmake

Reply:
I see my mistake. And this caused me issues when building, I ended up commenting out parameter callbacks. I see the format should look something like this: rclcpp_components_register_node(${PROJECT_NAME}

Good catch!